api/lib/domain/usecases/update-certification-center.js

```diff
@@ -25,29 +30,37 @@ async function _addOrUpdateDataProtectionOfficer({ certificationCenter, dataProt
 }
 
 const updateCertificationCenter = async function ({
-  certificationCenter,
+  certificationCenterId,
+  certificationCenterInformation,
   complementaryCertificationIds,
   certificationCenterForAdminRepository,
   complementaryCertificationHabilitationRepository,
   dataProtectionOfficerRepository,
+  centerRepository,
 }) {
-  certificationCenterCreationValidator.validate(certificationCenter);
+  certificationCenterCreationValidator.validate(certificationCenterInformation);
```

Comment:
Est-ce qu'il serait intéressant aussi de mettre à jour le validateur joi, avec quelque chose comme :

```js
isV3Pilot: Joi.boolean().when('isComplementaryAlonePilot', {
    switch: [{ is: true, then: Joi.valid(false) }],
    otherwise: Joi.forbidden(),
  }),
```

Reply:
Vu l'état du usecase (pas très propre) je trouve ça plus intéressant d'avoir la condition dans le usecase, on le voit mieux + on peut jeter un custom error.

Mais imaginons qu'on parte sur la validation Joi, ça demanderait bien plus de boulot non ?

Il faudrait que le modèle Center devienne celui qui gère cette validation pour avoir l'info du `isComplementaryAlonePilot` ?
Peut être que le refacto de ce usecase permettrait d'avoir ce genre de validation, autrement je laisserais tel quel

Reply:
Ouais carrément tu as raison